### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (#sidedocsframe;#back-label)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2342]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta#editor)
#### Element path
#sidedocsframe;#back-label
#### Snippet
`<span id="back-label">Go back</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.21 (foreground color: #aeaeae, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

``go back`` on side docs when button is disabled because there is no previously navigated page
